mimesniff/mime-types/charset-parameter.window.js

```diff
@@ -0,0 +1,27 @@
+promise_test(() => {
+  return fetch("resources/mime-types.json").then(res => res.json().then(runTests));
+}, "Loading data…");
+
+function runTests(tests) {
+  tests.forEach(val => {
+    if(typeof val === "string" || val.navigable !== true || typeof val.encoding !== "string") {
+      return;
+    }
+    const mime = val.input;
+    async_test(t => {
+      const frame = document.createElement("iframe"),
+            expectedEncoding = val.encoding === "" ? "UTF-8" : val.encoding;
```

Comment:
I think null, instead of the empty string, should be used to signify no parsed encoding.